resource/resource.go

```diff
@@ -204,7 +204,7 @@ type Actuator interface {
 //	if len(geometries) > 0 {
 //	   // Get the center of the first geometry
 //	   elem := geometries[0]
-//	   fmt.Println("Pose of the first geometry's center point:", elem.center)
+//	   fmt.Println("Pose of the first geometry's center point:", elem.Pose)
```

Comment:
Sorry I don't understand. "center" definitely seems to be wrong, and I think was just copied over from the Python SDK version of this. https://pkg.go.dev/go.viam.com/rdk@v0.42.0/spatialmath#Geometry seems relevant. Are you saying this should be 
```suggestion
//	   fmt.Println("Pose of the first geometry's center point:", elem.Pose())
```
? I'm having trouble testing since even after adding a box geometry to my fake arm in the frame config tab, len geometries = 0.

Reply:
Yeah the change in your suggestion is correct